shared/css/conversation.css

```diff
@@ -443,20 +406,131 @@ html, .fx-embedded, #main,
   height: 100%;
   right: 0;
   left: 0;
-  text-align: center;
   color: #000;
   z-index: 1010;
   display: flex;
   flex-flow: column nowrap;
-  justify-content: center;
   align-items: stretch;
 }
 
 .room-invitation-content {
   display: flex;
   flex-flow: column nowrap;
+  margin: 12px 0;
+  font-size: 1.4rem;
+}
+
+.room-invitation-content > * {
+  width: 100%;
+  margin: 0 15px;
+}
+
+.room-context-header {
+  font-weight: bold;
+  font-size: 1.6rem;
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+/* Input Button Combo group */
+.input-button-content {
+  margin: 0 15px 10px 15px;
+  min-width: 64px;
+  border-radius: 4px;
+  border: 1px solid #d2cece;;
+}
+
+.input-button-group-label {
+  color: #898a8a;
+  margin: 0 15px;
+  margin-bottom: 2px;
+  font-size: 1.2rem;
+}
+
+.input-button-content > * {
+  width: 100%;
+  padding: 0 4px;
+}
+
+.input-button-content .input-group input {
```

Comment:
Please change all the `.input-button-content` selectors to use child selectors (`>`), I think it won't break the layout if you just add the arrow in, and its more efficient.